multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -17,6 +18,7 @@ def default():
 @automm_presets.register()
 def medium_quality_faster_train():
     return {
+        "model.names": ["categorical_mlp", "numerical_mlp", "timm_image", "hf_text", "fusion_mlp"],
```

Comment:
Previously, clip was used in the `medium_quality_faster_train `.

Reply:
I added "clip" back. But I think we can remove it in 0.6. Currently, there is no clear advantage of fusing "clip" together with "timm" + "huggingface".

Reply:
I remember that adding clip helped improves the performances in image+text+tabular benchmarks. Will double check this.